components/docs-chef-io/content/habitat/accept-the-license.md

```diff
@@ -11,6 +11,7 @@ description = "Chef Habitat License"
     weight = 20
 
 +++
+[\[edit on GitHub\]](https://github.com/habitat-sh/habitat/blob/master/components/docs-chef-io/content/habitat/accept-the-license.md)
```

Comment:
We might want to remove this page entirely since we've already got the [Accepting the Chef License](https://docs.chef.io/chef_license_accept/#habitat) page in docs.chef.io.

Reply:
Good idea.